css/custom.css

```diff
@@ -4,7 +4,8 @@ nav.navbar {
 }
 
 footer.navbar {
-
+    background-color: #fff;
+    border: 1px solid rgba(0,0,0,.1);
```

Comment:
I think this looks better without a border.

With border...
<img width="1014" alt="screenshot 2016-07-22 16 59 10" src="https://cloud.githubusercontent.com/assets/624760/17063126/b3bd0a6a-502d-11e6-9627-0844a21ad903.png">

Without border...
<img width="977" alt="screenshot 2016-07-22 16 59 20" src="https://cloud.githubusercontent.com/assets/624760/17063127/b6570c8a-502d-11e6-8c76-e583c7fde855.png">

What do you think?

Note: I don't think the white background is needed on line 7.